File: app/src/main/java/com/marked/vifo/extra/HTTPStatusCodes.java
package com.marked.vifo.extra;

/**
 * Created by Tudor Pop on 22-Nov-15.
 */
public interface HTTPStatusCodes {
	String ERROR_RESPONSE_STATUS_CODE = "error_response";

	int NOT_FOUND = 404;
	int REQUEST_OK = 200;
	int REQUEST_TIMEOUT = 408;
	int REQUEST_CONFLICT = 409;

	int UNPROCESSABLE_ENTITY = 422;
}

File: app/src/main/java/com/marked/vifo/extra/HTTPStatusCodes.kt
package com.marked.vifo.extra

/**
 * Created by Tudor Pop on 22-Nov-15.
 */
interface HTTPStatusCodes {
	companion object {
		val ERROR_RESPONSE_STATUS_CODE = "error_response"

		val NOT_FOUND = 404
		val REQUEST_TIMEOUT = 408
		val REQUEST_CONFLICT = 409

		val REQUEST_INCORRECT_PASSWORD = 600
	}
}